tools/tune/crossover/crossover_gen_coefs.m

```diff
@@ -11,8 +11,8 @@
 end
 
 function crossover_2way = crossover_generate_2way(fs, fc);
-	crossover_2way.lp = [lp_iir(fs, fc, 0)];
-	crossover_2way.hp = [hp_iir(fs, fc, 0)];
+	crossover_2way.lp = (lp_iir(fs, fc, 0));
+	crossover_2way.hp = (hp_iir(fs, fc, 0));
```

Comment:
why need for () ?

Reply:
To Intend to specify expression precedence , I 've to use parentheses () rather than brackets [].

Reply:
You don't need here () for precedence, remove them? Since I assume lp/hp_iir() returns a matrix I think also the [] are unnecessary. [] is needed to make a scalar a matrix of 1x1 but I don't think this is such case.

Reply:
Yup, Thanks !!